# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/moz.build
# Commit: 20543d03a4cd
# Full Hash: 20543d03a4cda05c785011a25079aa42aec8b2ba
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2019-03-29 21:53:01
# Regressor Bug: 1494675
# File Overlap Count: 2
# Description:
#   Bug 1494675 - Remove AllocationHandle. r=padenot
#   
#   The handle was used to keep separate allocations of the same source in a single
#   process apart. Sources no longer use sharing, so we no longer need allocations
#   or their handles even as a concept.
# ==============================================================================

diff -r 9e7832fee028 -r 20543d03a4cd dom/media/webrtc/moz.build
--- a/dom/media/webrtc/moz.build	Wed Mar 27 14:10:27 2019 +0000
+++ b/dom/media/webrtc/moz.build	Wed Mar 27 14:10:24 2019 +0000
@@ -14,7 +14,6 @@
 XPIDL_MODULE = 'content_webrtc'
 
 EXPORTS += [
-    'AllocationHandle.h',
     'CubebDeviceEnumerator.h',
     'MediaEngine.h',
     'MediaEngineDefault.h',
